web/src/main/java/com/epam/rft/atsy/web/util/CandidateCVFileHandlerUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.rft.atsy.web.util;
+
+
+import com.epam.rft.atsy.service.domain.CandidateDTO;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+public class CandidateCVFileHandlerUtil {
+  private static final String SEPARATOR = "-";
+
+  private CandidateCVFileHandlerUtil() {}
+
+  public static File createCVFileFromFolderLocationAndCandidateDtoAndCVFilename(String folderLocation, CandidateDTO candidateDTO, String cvFilename) {
```

Comment:
Be careful with static methods, it is hard to mock static methods in unit tests. In this situation simply create a bean (for example with @Component annotation) instead of create class with only static methods.